### PR TITLE
Improve styling and auto-activate save button

### PR DESCRIPTION
**Pull request by blashca**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -141,6 +141,7 @@
     <string name="times">times a</string>
 
     <string name="record">Record</string>
```
Comment (hfossli) [inline app/src/main/res/values/strings.xml]: Is this "record" entry here excessive now?

Comment (blashca) [inline app/src/main/res/values/strings.xml]: Actually I use it in other activities as well.

**Comment by blashca** [comment]
Unless you have any objections I would merge it.

**Comment by Ehyeh-Asher-Ehyeh** [comment]
The code looks good to me, no objections

**Pull request merged**